### PR TITLE
Add column edit form

### PR DESCRIPTION
**Pull request by itkg-alavieille**

### CODE REVIEW

File: BackofficeBundle/Resources/public/coffee/areaFlex/AreaFlexToolbarView.coffee [hunk 1]
```diff
@@ -36,6 +38,16 @@ class OpenOrchestra.AreaFlex.AreaFlexToolbarView extends OrchestraView
     @updateToolbarPosition(@$el)
     $(window).bind 'scroll', () ->
       context.updateToolbarPosition(context.$el)
+  ###*
+   * Show form edit column
+  ###
+  showFormColumn: ->
+    console.log('show')
```
Comment (itkg-ngilain) [inline BackofficeBundle/Resources/public/coffee/areaFlex/AreaFlexToolbarView.coffee]: you should remove this one

File: BackofficeBundle/Resources/public/coffee/areaFlex/AreaFlexToolbarView.coffee [hunk 2]
```diff
@@ -36,6 +38,16 @@ class OpenOrchestra.AreaFlex.AreaFlexToolbarView extends OrchestraView
...
+    adminFormViewClass = appConfigurationView.getConfiguration(@options.entityType, 'showAdminForm')
+    new adminFormViewClass(
+      url: @options.area.get("links")._self_form_column
```
Comment (itkg-ngilain) [inline BackofficeBundle/Resources/public/coffee/areaFlex/AreaFlexToolbarView.coffee]: You should check that the link exists before building a view with it

File: BackofficeBundle/Resources/translations/messages.en.yml [hunk 3]
```diff
@@ -200,7 +200,13 @@ open_orchestra_backoffice:
             remove_sub: Remove
             block_exists: Can't add Area, area contains blocks
         area_flex:
+            success: The area has been modified
+            html_class: Html class for the area
+            label: Area label
+            area_id: The area id
```
Comment (itkg-ngilain) [inline BackofficeBundle/Resources/translations/messages.en.yml]: If it's the html id, you should mention it

File: BackofficeBundle/Resources/translations/messages.fr.yml [hunk 4]
```diff
@@ -185,7 +185,13 @@ open_orchestra_backoffice:
             yearly: Annuel
             never: Jamais
         area_flex:
+            success: La zone a été modifiée
+            area_id: Id de la zone
```
Comment (itkg-ngilain) [inline BackofficeBundle/Resources/translations/messages.fr.yml]: same about the id typology

**Comment by itkg-alavieille** [comment]
@itkg-ngilain pr upadted

**Comment by itkg-alavieille** [comment]
@itkg-ngilain no they haven't a limit, the area id can be larger  but in most cases, there will be two see three level and for the others cases yes this id will be long but not visible to the user.

**Comment by itkg-ngilain** [comment]
:+1:

**Pull request merged**